proto/cws/dumpsv1/activity_dump.proto

```diff
@@ -248,6 +256,37 @@ message BindNode {
     uint32 protocol = 5;
 }
 
+message NetworkDeviceNode {
+    repeated MatchedRule matched_rules = 1;
+
+    uint32 netns = 2;
+    uint32 ifindex = 3;
+    string ifname = 4;
```

Comment:
I'm not sure I see the value of these fields, why not putting the `flow_nodes` field directly in the `ProcessActivityNode`?

Reply:
Discussed offline thanks ! 👍🏻